### PR TITLE
[WIP] New package: protonmail-bridge-1.1.4

### PR DESCRIPTION
**Pull request by ArmedAviator**
I'm trying to get this package to install.  I did not work out all the details yet.  It downloads a binary .deb package, extracts the data and needs to copy it to /usr.

There seem to be alot of Qt libraries included.  I'm trying to get the maintainer at ProtonMail to provide source code and proper licensing.  Right now, it should NOT me merged to the repo, but I want to get this working to have more input for the developers at ProtonMail.

### CODE REVIEW

**Comment by maxice8** [comment]
The package seems proprietary, please set license to Proprietary and add restricted=yes, see `parsec` for an example on how to do it

**Comment by ArmedAviator** [comment]
Proprietary license and restricted changes have been pushed.

File: srcpkgs/protonmail-bridge/template [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+revision=1
+only_for_archs="x86_64"
+build_style=fetch
+short_desc="ProtonMail Bridge for use with E-mail software"
+maintainer="Rich G <rich@richgannon.net>"
+license="Proprietary"
+homepage="https://protonmail.com/bridge"
+distfiles="https://protonmail.com/download/beta/protonmail-bridge_${version}-1_amd64.deb"
+checksum=499fbb201dc8434ad8e3b401cd68373066065fed65dc2612568307e95b63e9d5
+restricted="yes"
```
Comment (maxice8) [inline srcpkgs/protonmail-bridge/template]: doesn't need to be quoted

**Comment by maxice8** [comment]
pleas add noverifyrdeps

File: srcpkgs/protonmail-bridge/template [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+build_style=fetch
+short_desc="ProtonMail Bridge for use with E-mail software"
+maintainer="Rich G <rich@richgannon.net>"
+license="Proprietary"
+homepage="https://protonmail.com/bridge"
+distfiles="https://protonmail.com/download/beta/protonmail-bridge_${version}-1_amd64.deb"
+checksum=499fbb201dc8434ad8e3b401cd68373066065fed65dc2612568307e95b63e9d5
+
+restricted=yes
+noverifyrdeps=yes
```
Comment (maxice8) [inline srcpkgs/protonmail-bridge/template]: nopie=yes too

Comment (ArmedAviator) [reply srcpkgs/protonmail-bridge/template]: Thanks maxice8.  The package now builds.  I have installed it on my local system and need to test it out and will give feedback to the devs.  Hopefully will return with a non-proprietary license and better support from them.

**Pull request closed without merging**